Project_Mars/Project_Mars/Pages/LanguageTab.cs

```diff
@@ -0,0 +1,270 @@
+﻿using NUnit.Framework;
+using OpenQA.Selenium;
+using OpenQA.Selenium.DevTools;
+using OpenQA.Selenium.DevTools.V117.Debugger;
+using OpenQA.Selenium.Support.UI;
+using Project_Mars.Utilities;
+using SeleniumExtras.WaitHelpers;
+using System.Data.SqlTypes;
+using System.Security.Cryptography.X509Certificates;
+using System.Threading;
+
+namespace Project_Mars.Pages
+{
+    public class LanguageTab
+    {
+        private readonly IWebDriver driver;
+
+        //public LanguageTab(IWebDriver driver)
+        //{
+        //    this.driver = driver;
+        //}
+
+        public LanguageTab()
+        {
```

Comment:
constructor not needed here as you have no instance variables being initialised, by default a default constructor is added on compile time